api/app/src/main/kotlin/packit/security/WebSecurityConfig.kt

```diff
@@ -53,6 +55,15 @@ class WebSecurityConfig(
         return httpSecurity.build()
     }
 
+    private fun getCorsConfigurationSource(): CorsConfigurationSource
+    {
+        val corsConfig = CorsConfiguration()
+        corsConfig.allowedOriginPatterns = listOf("http://localhost*", "https://localhost*")
```

Comment:
Should this be read from config rather than hardcoded? Is it going to be an issue when we deploy to non-local?